source/user-manual/reference/tools/keystore-tool.rst

```diff
@@ -0,0 +1,33 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+  :description: Tool where sensitive configuration data can be securely stored, including any information that the Wazuh server daemons/tools need for their work. 
```

Comment:
Wazuh server -> Wazuh manager

Wazuh server is Wazuh manager and Filebeat https://documentation.wazuh.com/current/getting-started/components/wazuh-server.html

Reply:
Done in [Wazuh components are named appropriately](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/ef178ca855e5c05eca6c5c8705a7e54e2199f3bb) commit.